AdyenActions/Actions/RedirectAction.swift

```diff
@@ -9,24 +9,64 @@ import Foundation
 /// Describes an action in which the user is redirected to a URL.
 public struct RedirectAction: Decodable {
     
+    /// Defines the type of redirect flow utilized by the `RedirectAction` object.
+    public enum RedirectType: String, Decodable {
+        // swiftlint:disable redundant_string_enum_value
+        case redirect = "redirect"
+        case nativeRedirect = "nativeRedirect"
+
+        public init(from decoder: Decoder) throws {
+            let container = try decoder.singleValueContainer()
+            let type = try container.decode(String.self)
+            self = RedirectType(rawValue: type) ?? .redirect
+        }
+    }
+
     /// The URL to which to redirect the user.
     public let url: URL
-    
+
     /// The server-generated payment data that should be submitted to the `/payments/details` endpoint.
     public let paymentData: String?
-    
+
+    /// Redirect type.
+    public let type: RedirectType
```

Comment:
Do we need to expose the property itself? We could keep it internal at the start

Reply:
Made it private